src/main/webapp/i18n/de/complaintResponse.json

```diff
@@ -1,5 +1,15 @@
 {
     "artemisApp": {
+        "locks": {
+            "acquired": "Du hast jetzt die Sperre auf diese Anfrage für 24 Stunden!",
+            "removeButton": "Entferne Sperre",
+            "lockRemoved": "Sperre von Anfrage entfernt",
+            "lockInformation": "Gesperrt von {{user}} bis {{endDate}}",
+            "lockInformationYou": "Gesperrt von DIR bis {{endDate}}",
+            "notUnlocked": "Nicht gesperrt",
+            "lockStatus": "Status der Sperre",
+            "notAllowedToRespond": "Es ist Dir nicht erlaubt auf diese Anfrage zu antworten!\nBei Team Aufgaben ist der Team Tutor sowohl für Beschwerden als auch Mehr-Feedback Anfragen verantwortlich\nBei individuellen Aufgaben werden Beschwerden von einem anderen Tutor beantwortet und mehr Feedback Anfragen von dem Tutor welcher die Aufgabe bewertet hat.\nBei Testläufen für Klausuren darf auch diese Tutor Beschwerden beantworten"
```

Comment:
This seems to be missing a fullstop after `verantwortlich`. (And at the end?)

Reply:
Fixed in d6f47cc2cae4dd19469ad53f94f841cb8f876b9a